ambry-protocol/src/test/java/com/github/ambry/protocol/RequestResponseTest.java

```diff
@@ -41,6 +41,7 @@
 import com.github.ambry.utils.SystemTime;
 import com.github.ambry.utils.TestUtils;
 import com.github.ambry.utils.Utils;
+import com.google.common.collect.Lists;
```

Comment:
Unused import

Reply:
@manbearpig1996 Fixed this, can you please check and approve it again?